src/components/Search/SearchPageHeader.tsx

```diff
@@ -56,13 +68,31 @@ function SearchPageHeader({
     const {activeWorkspaceID} = useActiveWorkspace();
     const {isSmallScreenWidth} = useResponsiveLayout();
     const {setSelectedTransactionIDs} = useSearchContext();
+    const {status} = queryJSON;
+
+    const subtitle = useMemo(() => {
+        if (!isSearchResultsMode) {
+            return '';
+        }
+
+        return translate('search.filtersHeader');
+    }, [isSearchResultsMode, translate]);
```

Comment:
using of `useMemo` is a great practice, but to me in here just for 1 call to `translate` I find its not really needed